projects/imagemagick.org/package.yml

```diff
@@ -0,0 +1,53 @@
+distributable:
+  #FIXME: they use a weird versioning scheme
+  url: https://github.com/ImageMagick/ImageMagick/archive/refs/tags/7.1.0-61.tar.gz
+  strip-components: 1
+
+versions:
+  #FIXME: they use a weird versioning scheme
+  # github: ImageMagick/ImageMagick
+  - 7.1.0.61
```

Comment:
@mxcl you ok with this for now? They use the pre-release identifier as a super-patch, or something.

Reply:
Fine for now, we will need to figure it out one day tho.